sla/index.md

```diff
@@ -86,4 +85,4 @@ This Service Level Agreement (“**SLA**”) is included for all customers that
     * Defects in the Tailscale Client Software due to device hardware malfunction, abuse or improper use; or
     * Any events beyond our reasonable control, including strikes or other labor problems, material shortages, epidemics, pandemics,  civil unrest, riots, insurrection, fires, flood, earthquakes, storms, explosions, cyber attacks, or acts of God, war, terror or government.
 
-    Finally, our ability to perform under this SLA is contingent upon Customer’s reasonable cooperation. If Customer has not provided all necessary information to Tailscale to resolve a reported issue, or Tailscale is otherwise denied or delayed access or information by Customer, then Tailscale will be excused, without liability, from performing under this SLA with respect to that issue. In such a case, Customer will not be entitled to any Service Credits, and any such failure to perform resulting from Customer’s failures will not relieve Customer from its payment obligations to Tailscale.
+    Finally, our ability to perform under this SLA may be contingent upon Customer’s reasonable cooperation. If Tailscale is denied or delayed access or information by Customer, as reasonably requested by Tailscale in connection with resolving reported issues, then Tailscale will be excused, without liability, from performing under this SLA with respect to that issue until such access or information is provided. In such a case, Customer will not be entitled to any Service Credits, nor will Customer from its payment obligations to Tailscale under the MSA.
```

Comment:
Missing words:

"nor will Customer from its payment obligations"

should be

"nor will Customer be excused from its payment obligations"